docs/api/ingest-budget-v1.md

```diff
@@ -34,9 +34,9 @@ import ApiIntro from '../reuse/api-intro.md';
 | FED        | https://api.fed.sumologic.com/docs/#tag/ingestBudgetManagementV1 |
 | IN         | https://api.in.sumologic.com/docs/#tag/ingestBudgetManagementV1  |
 | JP         | https://api.jp.sumologic.com/docs/#tag/ingestBudgetManagementV1  |
+| KR         | https://api.kr.sumologic.com/docs/#tag/ingestBudgetManagementV1  |
```

Comment:
V1 of this feature is going to be deprecated from Sept 16, 2024 so I think we should not list it on the help page.
cc @ajaiswals

Reply:
@vsinghal13 thank you so much for the heads up! I'll leave it in for now because I'm merging the PR today, and technically this endpoint still in effect through 9/16. We'll take this down when we pull down the rest of the ingest budget v1 content.